controller_manager:
  ros__parameters:
    update_rate: 100

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    joint_pid_controller:
      type: pid_controller/PidController


joint_pid_controller:
  ros__parameters:
    dof_names:
      - joint_1
      - joint_2
    command_interface: effort
    reference_and_state_interfaces: ["position"]
    gains: 
      joint_1: {p: 100.0, i: 0.01, d: 10.0, i_clamp_min: -0.5, i_clamp_max: 0.5}
      joint_2: {p: 100.0, i: 0.01, d: 10.0, i_clamp_min: -0.5, i_clamp_max: 0.5}
